### ISSUE TITLE
Attempting to sign up when email is already registered loses "personal device" checkbox

### ISSUE THREAD

**Issue by razzius** [razzius/hms-weave#129]
This whole flow could be reworked - should we just send the email?
We could have a useful message like "That email has already been registered. Check your email for a link to log in. In the future, you can log in directly by clicking "login" in the top navigation."

**Comment by JonKusner** [comment]
Yeah minimizing the email back and forth will be critical for student
involvement (who mostly view Weave on phones). I like the dialogue to tell
people that they are already registered. Is there a way to retain the
"personal device" status so that when they put in a recognized email it
just logs them right in?

**Comment by jjchen20** [comment]
It would be a helpful feature for students, so it's about the user email
and not the device that recognizes and keeps them logged in. For security
reasons though, I think we should keep recognition with the device and not
the email.

One concern is if people are accessing Weave on a hospital computer in the
workroom that may be on for long periods of time. That would be an issue if
students logged in and if the computers didn't log out afterward.

Faculty, for the most part, will have their own computers in their offices
so it would be less of a concern and could enhance fluidity if they access
Weave in the hospital and then want to edit their profile at home.

**Comment by razzius** [comment]
I'm going to implement this with an "already registered, check your email" approach.

**Comment by razzius** [comment]
@Chronowork1 this could be a good one for you to work on.
